### PR TITLE
Added a redirect for the DSTV Ontology, which represents an owl exten…

### PR DESCRIPTION
**Pull request by kirner-ip**
…sion of the existing DSTV standard for steel construction.

The ontology is part of the project Internet of Construction (see readme) and a future extension which is called BAUFEST 4.0

### CODE REVIEW

File: dstv/.htaccess [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Turn off MultiViews
+Options -MultiViews
+
+AddType text/turtle .ttl
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteRule ^$ https://ip.pages.rwth-aachen.de/ioc/dstv-ontologie/
+
+# If suffix ttl, redirect to turtle version
+RewriteRule ^$ https://ip.pages.rwth-aachen.de/ioc/dstv-ontologie/ontology.ttl
```
Comment (dgarijo) [inline dstv/.htaccess:13]: This will not work, as the previous rule superseeds it

File: dstv/.htaccess [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Default response: html
+RewriteRule ^$ https://ip.pages.rwth-aachen.de/ioc/dstv-ontologie/
```
Comment (dgarijo) [inline dstv/.htaccess:16]: same as above

**Comment by dgarijo** [comment]
@kirner-ip please have a look at my comments. If you add two rules that look the same, only the first one will trigger

File: dstv/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Interconnecting Data in Construction Processes - focus on steel construction and usage of the DSTV-NC standard
+See http://www.internet-of-construction.com/index_en.html
+
+## Contact
+This space is administered by:  
+
+**Lukas Kirner**  
+  *Researcher*  
+Individualized Production, RWTH Aachen, Germany.  
+https://github.com/kirner-ip
```
Comment (TallTed) [inline dstv/README.md:14]: The double-space-line-end is to preserve the line-break.
```suggestion
https://github.com/kirner-ip  
```

**Pull request closed without merging**